p11nethsm.example.conf

```diff
@@ -0,0 +1,46 @@
+# Set this option to true to enable the compatibility option for the C_SetAttributeValue() function.
+# This allows the applications using the Java Sun PKCS11 module (like EJBCA) to generate keys.
+# When using this, the names given to the keys will be ignored and the keys will have random names.
+# Under the hood it will store in memory the name given to the key when calling C_SetAttributeValue(). When a certificate is uploaded it will check if the name was previously passed to C_SetAttributeValue() and translate it to the real name on the NetHSM.
+enable_set_attribute_value: false
+
+# You can set the log file location here.
+# If no value is set the module will output to stderr.
+# If a value is set it will output to the file.
+log_file: /tmp/p11nethsm.log
+# Optional log level, acceptable  values are Trace, Debug, Info, Warn and Error
+log_level: Debug
+
+# Each "slot" represents a NetHSM server
```

Comment:
Actually, it’s a set of servers with the same users and keys (or at least a common subset), isn’t it?  Not sure if there is a preferred term for that setup.  Maybe “NetHSM cluster”?

Reply:
Yeah a slot represent a cluster of NetHSM that can be accessed with the same credentials.
https://docs.nitrokey.com/nethsm/pkcs11-setup#instances

Reply:
This was taken verbatim from [the docs](https://docs.nitrokey.com/nethsm/pkcs11-setup#configuration-file-format).
I Will fix it both here and there.

Reply:
Ah that's my bad. I should add that the slot name comes from PKCS#11 where it's like a card reader or HSM port.

Reply:
Once this is merged I will udpate it also in the docs.